ui/src/components/Rules/RuleGroup/AddModal/index.tsx

```diff
@@ -441,627 +432,677 @@ const AddModal = (props: AddModal) => {
-                  alt="BETA"
-                />
-              </label>
-              <div className="form-input">
-                <div className="form-input-field">
-                  <Button
-                    buttonType="default"
-                    type="button"
-                    name="notifications"
-                    onClick={() => {
-                      setConfigureNotificationModal(!configureNotificionModal)
-                    }}
-                  >
-                    <span>Configure</span>
-                  </Button>
+                    {tautulliEnabled && (
+                      <div className="flex flex-row items-center justify-between py-2 md:py-4">
+                        <label
+                          htmlFor="tautulli_watched_percent_override"
+                          className="text-label text-left"
+                        >
+                          Tautulli watched percent override
+                          <p className="text-xs font-normal">
+                            Overrides the configured Watched Percent in
+                            Tautulli, which is used to determine when media is
+                            counted as watched
+                          </p>
+                        </label>
+                        <div className="flex justify-end px-2 py-2">
+                          <div className="form-input-field w-20">
+                            <input
+                              type="number"
+                              min={0}
+                              max={100}
+                              name="tautulli_watched_percent_override"
+                              id="tautulli_watched_percent_override"
+                              defaultValue={
+                                collection
+                                  ? collection.tautulliWatchedPercentOverride
+                                  : ''
+                              }
+                              ref={tautulliWatchedPercentOverrideRef}
+                            />
+                          </div>
+                        </div>
+                      </div>
+                    )}
+                  </div>
                 </div>
               </div>
             </div>
-
-            <hr className="mt-5" />
-            <div className={`section ${useRules ? `` : `hidden`}`}>
-              <div className="section">
-                <div className="max-width-form-head flex">
-                  <div className="ml-0">
-                    <h3 className="heading">Rules</h3>
-                    <p className="description">
-                      Specify the rules this group needs to enforce
-                    </p>
+          </div>
+          <hr className="mt-6 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+          <div className="grid grid-cols-1">
+            <div className="flex justify-center">
+              <div className={`section ${useRules ? `` : `hidden`} md:w-3/4`}>
```

Comment:
We can pick this up in a future PR, but I think we could keep the rules section here full width until a larger breakpoint like XL. Some of the rule names are quite long so this change would allow the user to see more of it. I did this locally and it looks like some other styling needs changing elsewhere, hence I think we can pick this up later.